media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -8,8 +8,10 @@ <h2>
 údajov
 </h2>
 <p>
-podľa zákona č. 122/2013 Z.z. o ochrane osobných údajov a o zmene a doplnení niektorých zákonov
-v platnom znení (ďalej ako „Zákon“)
+podľa zákona č.
+<a href="https://www.slov-lex.sk/pravne-predpisy/SK/ZZ/2013/122/?ucinnost=01.12.2015" target="_blank">122/2013 Z.z.</a>
```

Comment:
Predpis 122/2013 bol zrušený predpisom 18/2018 Z. z.. V záhlaví stránky je uvedený rok 2015, Preto som použil tento dátum.

Reply:
@mmmaly Je to OK, ze sa odvolavame na zruseny zakon?

Reply:
dajme tam číslo noveho zakona 18/2018 Z. z.  (  https://www.slov-lex.sk/pravne-predpisy/SK/ZZ/2018/18/20190901 )